Applications/Telemetry/readme.md

```diff
@@ -47,17 +51,68 @@ There are two ways to start Telemetry in Windows. First by command line and seco
+
+### Web Browser
+
+Using the RESTful Flask server enables development with web tools. We can query Telemetry for data from any web browser on your local computer. You also can use any of the development tools included with your browser to develop applications.
+
+NOTE: Please see the Telemetry API section below for more details on what can be queried!
+        
+#### Example Web Queries
+
+Telemetry expects several input URL parameters to perform a query on the SQLite database. If your query returns no results you will receive an HTTP 204 response which may result in your browser simply showing you the current page which could be stale data so please be aware. We also attempt to provide useful error feedback in JSON response.
+
+##### Return Last Five Minutes
+At a minimum to perform a Telemetry query you can simply return the last five minutes of telemetry data in the entire database
+
+Query: `http://localhost:8001/`
+
+This results in a JSON response which can be viewed in the browser.
+
+![Telemetry Full Browser Response](images/WebBrowser_FullOutput.exe.png)
+
+##### Return Specific Station Data From Last Five Minutes
+This response requires a specific `callsign` and `nodeid` to be search for as a URL parameter. This will respond with telemetry specific to ONLY the requested radio, local or remote, heard in the last five minutes.
+
+Query: `http://localhost:8001/?callsign=kb1lqc&nodeid=1`
+
+![Telemetry Specific Station Browser Response](images/BrowserTelemetry_SpecificStation.png)
+
+##### Return Limited Number of Telemetry Items
+The `limit` URL parameter can be used with any of the URL search queries to limit the response to a specific number of telemetry items. If i wanted to limit the response from a query for KB1LQC-1 telemetry items to just one item I would perform the following query.
+
+Query: `http://localhost:8001/?callsign=kb1lqc&nodeid=1&limit=1`
+
+![Telemetry Specific Station Browser Response Limit](images/BrowserTelemetry_Limit.png)
+
+### SB Browser for SQLite
+#### Windows
+
+#### Linux (Debian-based)
 The DB Browser website has clear and concise instructions to install the browser. However, one tip will be to install `apt-get-repository`. It is located in the `software-properties-common` package.
 
  * `sudo apt-get install software-properties-common`
 
+#### Mac OSX
+
+## Telemetry API
+
+The following URL parameters are possible when interfacing the Telemetry server via a HTTP GET query, uppercase is not required for web URL input:
+
+ * `CALLSIGN`: Station callsign
+ * `NODEID`: Station node ID
+ * `DIRECTION`: Boolean 0 or 1 indicating return of only local or RF station data respectively
+ * `STARTTIME`: Start time (ISO 8601 "%Y-%m-%dT%H:%M:%S") of data to be obtained from SQLite database
+ * `ENDTIME`: End time (ISO 8601 "%Y-%m-%dT%H:%M:%S") of data to be obtained from SQLite database
+ * `TIMESPAN`: Timespan ending with current time of data to be obtained from SQLite database
```

Comment:
From reading this I have no idea what timespan is, can I just say "...&timespan=15"? Is that 15 seconds or minutes or days?